packages/pigeon/lib/dart_generator.dart

```diff
@@ -24,18 +25,30 @@ const String _standardMessageCodec = 'StandardMessageCodec';
 /// Options that control how Dart code will be generated.
 class DartOptions {
   /// Constructor for DartOptions.
-  const DartOptions({this.copyrightHeader});
+  DartOptions({
+    this.copyrightHeader,
+    this.dartOutPath,
```

Comment:
`Path` is inconsistent with the same field in the C++ options; all of them should either use the `Path` suffix or not, consistently. (I would vote with, for added clarity).

`dart` is redundant with the struture, and doesn't actually distinguish from the test code which is also Dart; I would call it `sourceOutPath`.